ferrocene/tools/traceability-matrix/README.md

```diff
@@ -17,6 +17,8 @@ On the one hand, we have the Ferrocene Language Specification (FLS). Simply spea
 
 On the other hand we have tests. We need to find at least one test for each section from the FLS. This test is then annotated with the id of the statement. This annotating is happening through a comment which is getting added to the end of the file. It is possible to annotate one test with multiple ids.
 
+Currently we annotate tests in `tests/ui` (for the FLS) and `tests/run-make` (for the command-line interface).
```

Comment:
there is also `tests/ui/ferrocene/compiler-arguments`

Reply:
They are for the command-line interface as well, are they not?

Reply:
Also, what does the `um` in the command-line interface ids (e.g. `um_rustc_cap_lints`) stand for?

Reply:
User Manual

Reply:
ahh